hail/python/hail/vds/methods.py

```diff
@@ -642,7 +642,14 @@ def split_multi(vds: 'VariantDataset', *, filter_changed_loci: bool = False) ->
     :class:`.VariantDataset`
     """
     variant_data = hl.experimental.sparse_split_multi(vds.variant_data, filter_changed_loci=filter_changed_loci)
-    return VariantDataset(vds.reference_data, variant_data)
+    reference_data = vds.reference_data
+
+    if 'GT' in reference_data.entry and 'LGT' in reference_data.entry:
+        reference_data = reference_data.drop('LGT')
+    elif 'LGT' in reference_data.entry:
+        reference_data = reference_data.transmute_entries(GT=reference_data.LGT)
```

Comment:
Could you factor 'LGT' like:
```python
if 'LGT' in reference_data.entries:
   # branch on if 'GT' exists
```
Also, would `reference_data` still contain 'LGT' after 'transmute_entries'?

Reply:
Reference data will not contain LGT as transmute methods work similar to `mt.annotate(foo=mt.bar).drop('bar')`.